### PR TITLE
Docs for Running Multiplayer Enabled Projects

### PR DESCRIPTION
**Pull request by AMZN-puvvadar**
## Change summary

Provides documentation addressing https://github.com/o3de/o3de.org/issues/1673

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Multiplayer projects can be [run in Editor](./test-in-editor.md) using Ctrl-G. See the accompanying documentation for more detail. 
+
+## Running locally using ImGui Options
+
+1. Launch your `ServerLauncher` either from Visual Studios or from your build directory.
+2. Press the `HOME` key to enable the debug menu.
+3. Select the `Host` option.
+4. Select a level to load to begin hosting.
+4. Use the `Launch Local Client` option to automatically launch a client connected to the server.
```
Comment (AMZN-Gene) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:23]: 4 -> 5

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Running Server and Client Launchers and configuring via console 
+
+You can manually launch the executables for Client and Server and configure them using the application's console command line.
+
+1. Launch both `ClientLauncher` and `ServerLauncher`
+2. In `ServerLauncher` press `~` to open the command line.
+3. Run the command `host` to begin hosting.
+4. Run the command `LoadLevel <path to level>` to load a level.
+5. In `ClientLauncher` press `~` to pen the command line.
```
Comment (AMZN-Gene) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:33]: pen -> open

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+6. Run the command `connect <IP Address>` to connect to the server. If running locally, `connect` will default to localhost.
+
+## Running Server and Client Launchers and configuring via cvar file
+
+You can manually launch the executables for Client and Server and pass a pre-defined configuration file to each for them to execute on launch. Commands in these cfgs files are executed in listed order.
```
Comment (AMZN-Gene) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:38]: +1 for calling out "executed in listed order". I remember that being questioned during workflow tests :)

**Review by AMZN-Gene — APPROVED** [review]

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
+* Build your project's `GameLauncher` and `ServerLauncher`
+* Use one of the following methods to run
+
+## In Editor Testing
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:13]: Sentence casing as per style guide for docs. "Testing in the Editor" maybe?

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:9]: Does this cover how to add and build the Multiplayer gem to you project?

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+* Build your project's `GameLauncher` and `ServerLauncher`
+* Use one of the following methods to run
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:11]: nit: as this is not a list maybe?

```suggestion
* Use run the game either in standalone mode, using the ImGui options to host, or via the in editor testing approach.
```

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
+---
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
+* Build your project's `GameLauncher` and `ServerLauncher`
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:10]:
```suggestion
* Build your project's `GameLauncher` and `ServerLauncher` targets.
```

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+## Running locally using ImGui Options
+
+1. Launch your `ServerLauncher` either from Visual Studios or from your build directory.
+2. Press the `HOME` key to enable the debug menu.
+3. Select the `Host` option.
+4. Select a level to load to begin hosting.
+5. Use the `Launch Local Client` option to automatically launch a client connected to the server.
+
+## Running Server and Client Launchers and configuring via console
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:25]: Maybe split into?

## Running Server and Client launcher in Standalone

### Manual configuration using the application's console

### Using pre-defined config files

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+1. In your project directory create `launch_client.cfg` and `launch_server.cfg`.
+2. Open `launch_server.cfg` and edit it to look like the following:
+```
+host
+LoadLevel <path to Level>
+```
+3. Open `launch_client.cfg` and edit it to look like the following:
+```
+connect <IP Address>
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:48]: connect \<IP address:Port\> ?

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+```
+You can alternatively do the following for testing locally:
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:50]:
```suggestion
You can alternatively do the following for testing against localhost server running on the default port.
```

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+```
+connect
+```
+4. Run the server using `MultiplayerSample.ServerLauncher.exe --console-command-file=launch_server.cfg`
+5. Run the client using `MultiplayerSample.ClientLauncher.exe --console-command-file=launch_client.cfg`
+
+It is necessary to run the Server first so the Client has a host to connect to. These can be run either from command line in your build directory or by setting command line arguments in your preferred execution method.
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:57]:
```suggestion
It is necessary to run the Server first so the Client has a host to connect to. 

The commands above can be run either from command line in your build directory or by setting command line arguments in your preferred execution method.
```

**Review by lmbr-pip — COMMENTED** [review]
LG

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md (hunk 5 above)
Comment (AMZN-puvvadar) [reply content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:9]: Yep

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
+* Build your project's `GameLauncher` and `ServerLauncher` targets.
+* Use run the game either in standalone mode, using the ImGui options to host, or via the in editor testing approach.
```
Comment (lmbr-pip) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:11]:
```suggestion
* Run the game either in standalone mode using the ImGui options to host, or via the in editor testing approach.
```

**Review by lmbr-pip — APPROVED** [review]
One minor comment, looks good otherwise.

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
+---
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:9]:
```suggestion
* Ensure your project is [properly configured](./configuration.md).
```

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+1. Launch your `ServerLauncher` either from Visual Studios or from your build directory.
+2. Press the `HOME` key to enable the debug menu.
+3. Select the `Host` option.
+4. Select a level to load to begin hosting.
+5. Use the `Launch Local Client` option to automatically launch a client connected to the server.
+
+## Running Server and Client launchers in Standalone
+
+### Manual configuration via the application's console
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:27]:
```suggestion
### Manual configuration using the application's console 
```

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+4. Select a level to load to begin hosting.
+5. Use the `Launch Local Client` option to automatically launch a client connected to the server.
+
+## Running Server and Client launchers in Standalone
+
+### Manual configuration via the application's console 
+
+You can manually launch the executables for Client and Server and configure them using the application's console command line.
+
+1. Launch both `ClientLauncher` and `ServerLauncher`
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:31]:
```suggestion
1. Launch both `ClientLauncher` and `ServerLauncher`.
```

File: content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+* Build your project's `GameLauncher` and `ServerLauncher` targets.
+* Run the game either in standalone mode using the ImGui options to host, or via the in editor testing approach.
+
+## Testing in the Editor
+
+Multiplayer projects can be [run in Editor](./test-in-editor.md) using Ctrl-G. See the accompanying documentation for more detail. 
+
+## Running locally using ImGui Options
+
+1. Launch your `ServerLauncher` either from Visual Studios or from your build directory.
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md:19]:
```suggestion
1. Launch your `ServerLauncher` either from Visual Studio or from your build directory.
```

**Pull request merged**